ui/core/components/suggest_reforges_action.tsx

```diff
@@ -525,24 +509,26 @@ export class ReforgeOptimizer {
 									values: [
 										{ name: 'Select preset', value: 0 },
 										...[...statPresets.keys()].map(key => {
-											const ratingValue = statPresets.get(key)!;
-											const percentOrPointsValue = unitStat.convertRatingToPercent(ratingValue)!;
-											const percentValue = unitStat.equalsStat(Stat.StatMasteryRating) ? (percentOrPointsValue * this.player.getMasteryPerPointModifier()) : percentOrPointsValue;
+											const percentOrPointsValue = statPresets.get(key)!;
+											const percentValue = unitStat.equalsStat(Stat.StatMasteryRating)
+												? percentOrPointsValue * this.player.getMasteryPerPointModifier()
+												: percentOrPointsValue;
+
 											return {
 												name: `${key} - ${percentValue.toFixed(2)}%`,
-												value: ratingValue,
+												value: percentOrPointsValue,
 											};
 										}),
 									].sort((a, b) => a.value - b.value),
 									enableWhen: () => this.isAllowedToOverrideStatCaps || !this.softCapsConfig.some(config => config.unitStat.equals(unitStat)),
 									getValue: () => {
-										let ratingValue = unitStat.convertDefaultUnitsToRating(this.statCaps.getUnitStat(unitStat))!;
+										let ratingValue = this.statCaps.getUnitStat(unitStat);
```

Comment:
I don't think this is correct, it will break for Expertise. You need to call `convertDefaultUnitsToPercent()` so that it will handle unit conversions automatically. Also rename the variable to `percentValue` because it's incorrect and very confusing currently.